docs/reference/analysis/tokenfilters/keyword-marker-tokenfilter.asciidoc

```diff
@@ -1,137 +1,189 @@
 [[analysis-keyword-marker-tokenfilter]]
-=== Keyword Marker Token Filter
+=== Keyword marker token filter
+++++
+<titleabbrev>Keyword marker</titleabbrev>
+++++
 
-Protects words from being modified by stemmers. Must be placed before
-any stemming filters.
+Excludes specified tokens, called <<stemmer-keyword,keywords>>, from
+<<stemming,stemming>>.
 
-[cols="<,<",options="header",]
-|=======================================================================
-|Setting |Description
-|`keywords` |A list of words to use.
+The `keyword_marker` filter is often used with custom
+<<analysis-lang-analyzer,language analyzers>> to protect specific words from
+stemming. This filter uses Lucene's
```

Comment:
This seems redundant. Is the point that you need to use a create a custom analyzer to use this token filter?

Reply:
No. Keywords are commonly incorporated into custom _language_ analyzers, like shown in these examples:
https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis-lang-analyzer.html#arabic-analyzer

However, that's not super important here so I removed it with b53322d6f4d.